### ISSUE TITLE
Deep learning in 2016 (Karapathy's tweet)

### ISSUE THREAD

**Issue by sotetsuk** [sotetsuk/machine-learning-survey#15]
https://twitter.com/karpathy/status/709465955223543808

> BatchNorm, STN, DCGAN, DRAW, soft/hard attention, char-rnn, DeepDream, NeuralStyle, TensorFlow, ResNet, AlphaGo.. a lot happened over 1 year
